### PR TITLE
Dynamic Endpoint Sample | Mediation Extensions

### PR DESCRIPTION
**Pull request by ruthryi**
A -  808

### CODE REVIEW

File: en/docs/learn/api-gateway/message-mediation/adding-dynamic-endpoints.md [hunk 1]
```diff
@@ -17,8 +17,15 @@ The default endpoint sends the message to the address specified in the **To** he
 In this example, you have constructed the `service_ep` property dynamically and assigned the value of this property to the **To** header. The default endpoint sends the message to the address specified in the **To** header, in this case, 
 `http://jsonplaceholder.typicode.com/posts/`. 
 
+!!! info
+    The dynamic endpoint functionality is suitable for scenarios where the application client can send an attribute in the request correlating to the intended endpoint (such as an HTTP transport header or as part of the payload) which can be used in the mediation extension.
```
Comment (samgnaniah) [inline en/docs/learn/api-gateway/message-mediation/adding-dynamic-endpoints.md:21]:
```suggestion
    The dynamic endpoint functionality is suitable for scenarios where the application client can send an attribute in the request correlating to the intended endpoint (such as an HTTP transport header or as part of the payload), which can be used in the mediation extension.
```

File: en/docs/learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md [hunk 2]
```diff
@@ -34,9 +34,10 @@ Following is a manually created sample custom mediation policy. This custom sequ
       </log>
     </sequence>
     ```
+
 Follow the below steps to upload the above sequence as a custom mediation policy in the **Request** message flow.
 
-1.  You can copy the above mediation sequence into a **XML** file.
+1. You can copy the above mediation sequence into a **XML** file.
```
Comment (samgnaniah) [inline en/docs/learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md:40]:
```suggestion
1. You can copy the above mediation sequence into an **XML** file.
```

File: en/docs/learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md [hunk 3]
```diff
@@ -85,8 +86,8 @@ If you want to edit an already attached mediation policy,
 
 If you want to dis-engage any mediation policy that is already engaged,
 
-1.  Go to the Edit option in the **Message Mediation** section.    
-You can do this for Request, Response and/or Fault message flows.    
+1.  Go to the Edit option in the **Message Mediation** section.
```
Comment (samgnaniah) [inline en/docs/learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md:89]:
```suggestion
1.  Go to the **Edit** option in the **Message Mediation** section.
```

File: en/docs/learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md [hunk 4]
```diff
@@ -147,6 +148,9 @@ You can copy this content into an XML file (e.g., `global_ext.xml` ) and save it
 !!! tip
     If you are having a distributed setup, do the changes in **Gateway** node.
 
+!!! note
+    The mediation extension is applied to all resources of the API. Therefore, all resources will contain a similar logic to construct the endpoint.
```
Comment (lkokila) [inline en/docs/learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md:152]: @ruthryi  This document contains details about mediation sequences in general. Let's remove the "Therefore, all resources will contain a similar logic to construct the endpoint." part here.

**Review by lkokila — CHANGES_REQUESTED** [review]

**Review by samgnaniah — APPROVED** [review]

**Pull request merged**